src/Microsoft.AspNet.Identity.EntityFramework/IdentityEntityFrameworkServices.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.AspNet.Identity
     /// <summary>
     /// Default services
     /// </summary>
-    public class IdentityEntityFrameworkServices
+    internal class IdentityEntityFrameworkServices
```

Comment:
Hence why I asked him to make it internal, we got rid of the public pattern yes.  But we call this code in two places (with a different key type), so its useful as a helper method.  I can move it into the Builder extension class as a private instead, but the idea was to make this go away from a public API perspective